libnavigation-core/src/main/java/com/mapbox/navigation/core/fasterroute/FasterRouteController.kt

```diff
@@ -43,6 +43,7 @@ internal class FasterRouteController(
     }
 
     fun stop() {
+        // TODO add cancellation of the request
```

Comment:
nit : remove or handle TODO?

Reply:
Planning to cut a follow-up ticket - this wasn't handled before the contribution either.